services/src/main/java/com/linkedin/kmf/services/TopicManagementService.java

```diff
@@ -113,6 +116,14 @@ boolean someBrokerWithoutLeader() {
     }
 
     /**
+     *
+     * @return true if the current topic replication factor is less than the expected ratio
+     */
+    boolean insufficientReplicationFactor() {
+      return replicationFactor() < _topicReplicationFactor;
```

Comment:
_topicReplicationFactor might be better called _configuredReplicationFactor.  replicationFactor() can then be renamed something like currentReplicationFactor().